### ISSUE TITLE
[ADD]: new vult share name

### ISSUE THREAD

**Issue by jpthor** [vultisig/vultisig-ios#1134]
Currently vault share backups are a little confusing

simplify:
```
{name}-{XXXX}-part{t}of{n}.vult

XXXX - last 4 digits of pubkey

part1of2
part2of3

assign as per order in the keygen. 
```

**Comment by johnnyluo** [comment]
Let's update the backup file's name to make sure it follows this new pattern
1. n is the total number of `signer` field
2. t is the position of local party in signers array , start from 1

**Comment by Rockindash** [comment]
Also add a central view model to handle all the backup extensions.

**Comment by johnnyluo** [comment]
ensure `import vault` screen can support `.bak`,`.vult`,`.dat`, and original .json file

**Comment by johnnyluo** [comment]
closing fixed at #1212
